src/pre_defined_classes/mathematical_program/hard/q1/JavademicTest.java

```diff
@@ -0,0 +1,23 @@
+package pre_defined_classes.mathematical_program.hard.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.IntegerLiteral;
+import global.variables.StringLiteral;
+import pre_defined_classes.mathematical_program.hard.q1.Javademic;
+
+public class JavademicTest extends BaseTest {
+
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("Num cases of infections in "),
+                new IntegerLiteral(30),
+                new StringLiteral(" days is "),
+                new StringLiteral("13099.97821824974"),
```

Comment:
I think this should also use DoubleLiteral so we can validate the math, but it's fine for now - make an issue so we can revisit this later

Reply:
Yeah, it definitely should, but I was just working with what we have for now.